includes/fbutils.php

```diff
@@ -34,6 +34,9 @@ class WC_Facebookcommerce_Utils {
 		const FB_VARIANT_PATTERN = 'pattern';
 		const FB_VARIANT_GENDER  = 'gender';
 
+		const FB_DESCRIPTION = 'fb_description';
+		const WOO_DESCRIPTION = 'woo_description';
```

Comment:
Lets avoid prefix like 'Woo' in our plugin codebase. Either use the WooCommerce's 'description' or use prefix like 'wc' only if it represents WooCommerce's value for description attribute